spec/factories/attendances.rb

```diff
@@ -1,6 +1,14 @@
 FactoryGirl.define do
   factory :attendance do
```

Comment:
Could you on the way please also fix the indentation for this factory? It's kinda hard to understand like this.